test/integration/features/cmd-addons.feature

```diff
@@ -8,12 +8,9 @@ Feature: Addons command and its subcommands
       And executing "minishift add-ons" fails
       And executing "minishift add-on" fails
 
- Scenario: Installing default add-ons manually
```

Comment:
Instead of removing this scenario, you should keep it under the ' Default add-ons are installed but not enabled by default' scenario maybe once w/ and w/o the '--force' option. By removing it you remove the manual install test compleltey.

Reply:
@hferentschik nope. The manual install test is there at the line number 50 of this file. I changed it as part of https://github.com/minishift/minishift/pull/1419/ and now I reverted it back.